properties-files/openshiftv4_scale.properties

```diff
@@ -0,0 +1,24 @@
+CLUSTER_USER=kubeadmin
+CLUSTER_PASSWORD=
+CLUSTER_API_URL=
+SSHKEY_TOKEN=
+ORCHESTRATION_HOST=localhost
+ORCHESTRATION_USER=root
+WORKLOAD_IMAGE=quay.io/openshift-scale/scale-ci-workload
+WORKLOAD_JOB_NODE_SELECTOR=false
+WORKLOAD_JOB_TAINT=false
+WORKLOAD_JOB_PRIVILEGED=false
+KUBECONFIG_FILE=~/.kube/config
+PBENCH_INSTRUMENTATION=false
+ENABLE_PBENCH_AGENTS=false
+ENABLE_PBENCH_COPY=false
+PBENCH_SSH_PRIVATE_KEY_FILE=~/.ssh/id_rsa
+PBENCH_SSH_PUBLIC_KEY_FILE=~/.ssh/id_rsa.pub
```

Comment:
These two files are not needed in the properties file since they are provided build portion of the jenkins job.